src/Setup/WixBA/WixBA.cs

```diff
@@ -108,12 +142,22 @@ public static void PlanLayout()
                 WixBA.Plan(WixBA.Model.Command.Action);
             }
         }
+        /// <summary>
+        /// Helper to attach a debugger to the cr instance when debugging
+        /// </summary>
+        [Conditional("DEBUG")]
+        private static void AttachDebuger()
+        {
+            System.Diagnostics.Debugger.Launch();
+            return;
+        }
 
         /// <summary>
         /// Thread entry point for WiX Toolset UX.
         /// </summary>
         protected override void Run()
         {
+            // AttachDebuger();
```

Comment:
When I'm debugging a BA, I use Rob's suggestion of setting the `burn.clean.room` parameter to the path of the bundle so it doesn't actually create a clean room process.  That let me use my original process - set the BA project as the startup project and configure the project to launch the bundle for debugging.